### ISSUE TITLE
🚧 Dev ➾ Automated tests for Google Analtyics tracking

### ISSUE THREAD

**Issue by alexzbusko** [ecadlabs/taqueria#1456]
### 🚥 Definition of Done

- [ ] 🎯 Planning and Design
- [ ] 🏎️ Dev Implementation Complete
- [ ] ♾️ PR Merged
- [ ] 🏆 Product Acceptance

### ✅ To Do

Create the following tests for the plugins:
Google Analytics:

- opt-in
-- Success
-- Cancel
- opt-out
-- cancel
-- success

### ⚖️ Acceptance Criteria

✨ Functional: 
- [ ] 
- [ ] 

### 🔭 Scope

🙅‍♂️ Out of Scope: 
- ... 
 
 

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by hu3man** [comment]
Currently blocked by `-y` bug #1455
